modules/Microsoft.Insights/webTests/deploy.bicep

```
@@ -0,0 +1,144 @@
+@description('Required. Name of the webtest.')
+param name string
+
+@description('Required. User defined name if this WebTest.')
+param webTestName string
+
+@description('Optional. Tags of the resource.')
+param tags object = {}
+
+@description('Required. The collection of request properties.')
+param request object
+
+@description('Optional. Location for all Resources.')
+param location string = resourceGroup().location
+
+@description('Optional. User defined description for this WebTest.')
+param descriptionWebTest string = ''
```

I'd suggest to rename it `description`. Then replace all metadata `@description` with `@sys.description` (like done for example in the nested_roleAssignments template)

---

updated as suggested

---

Only half, actually. The other half would require
```suggestion
param description string = ''
```
Along side the corresponding changes in the other locations